packages/notifier/test/test-makeNotifierFromSubscriber.js

```diff
@@ -45,7 +49,7 @@ test('makeNotifierFromSubscriber(finishes) - for-await-of iteration', async t =>
     conclusionMethod: 'finish',
     conclusionValue: 'done',
   });
-  const notifier = await makeNotifierFromSubscriber(subscriber);
```

Comment:
curious why these were `await` before. `makeNotifierFromSubscriber` is sync in master.

Reply:
The `await`s were unnecessary both before and after this PR.